### PR TITLE
Post practice session report

### PR DESCRIPTION
**Pull request by milindvishnoi**

### CODE REVIEW

File: packages/app/src/pages/PostPracticeSession.vue [hunk 1]
```diff
@@ -27,36 +27,55 @@
...
+    AnswerDescription,
+  },
 };
 </script>
 
 <style scoped>
 .mobile-container {
   width: 420px;
-  max-height: 85%;
+  min-height: 85%;
```
Comment (ArsalaBangash) [inline packages/app/src/pages/PostPracticeSession.vue:70]: What were you trying to fix with this?

File: packages/app/src/store/practice/actions.ts [hunk 2]
```diff
@@ -107,6 +96,15 @@ export const actions: ActionTree<PracticeState, any> = {
     context.commit(PracticeMutations.RESET_PRACTICE_SESSION)
   },
   skipQuestion(context) {
+    const questionReport = {
+        question: context.state.question.infix,
+        answer: "-",
```
Comment (ArsalaBangash) [inline packages/app/src/store/practice/actions.ts:101]: Why not an empty string here?

File: packages/app/src/store/practice/mutations.ts [hunk 3]
```diff
@@ -85,9 +64,15 @@ export const mutations: MutationTree<PracticeState> = {
     state.practiceLastQuestionCorrect = practiceLastQuestionCorrect;
   },
   addPracticeAttemptedQuestion(state: PracticeState, payload: QuestionReport) {
-    state.practiceAttemptedQuestions.questions.push(payload)
-  },
-  createQuestionReport(state: PracticeState) {
-    
+    var added = false;
```
Comment (ArsalaBangash) [inline packages/app/src/store/practice/mutations.ts:67]: Use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find for more concise code

**Comment by milindvishnoi** [comment]
PR Notes
- [] Change implementation for wrong Q's
- [] Add Color Plate
- [] Add timer mode for all Q's
- [] Reset after every play session

**Pull request closed without merging**